packages/astro/src/compiler/transform/prism.ts

```diff
@@ -11,14 +11,14 @@ function escape(code: string) {
     .replace(/[`$]/g, (match) => {
       return '\\' + match;
     })
-    .replace(/&#123;/g, '{');
+    .replace(/CURLY_BRACE/g, '{');
```

Comment:
where does `CURLY_BRACE` come from? I don't see this getting created anywhere, is it meant to be `LEFT_CURLY_BRACKET_FIX` instead?

Reply:
Yes. Done in c094059.